### DISCUSSION TITLE
Ubuntu 21.10 and fullscreen freerdp doesn't display mouse pointer

### DISCUSSION THREAD

**Discussion by mkoren** [FreeRDP/FreeRDP#7378] [General]
When creating a full-screen session to a Windows remote client, the mouse pointer on the Windows remote client isn't displayed. The window is receiving the mouse input (I can see things in the window that have a mouseover event get highlighted if I accidentally move the mouse over it, I just don't see the actual mouse cursor the depicts the mouse cursor location).

This first started after I upgraded to Ubuntu 21.10 from Ubuntu 20.04. I have re-installed freerdp using:

sudo apt remove freerdp2-x11 -y
sudo apt remove freerdp2-shadow-x11 -y

sudo apt install freerdp2-x11 -y
sudo apt install freerdp2-shadow-x11 -y

The command I'm using to connect to the Windows remote client is:

xfreerdp /bpp:32 /u:username /p:password +fonts +window-drag /floatbar:show:always /f /t:"Windows File Server" /v:192.168.0.15

The above command worked as expected prior to upgrading to Ubuntu 21.10.

Mark Korenkiewicz

**Comment by akallabeth** [comment]
Sounds like an ubuntu specific issue.
1. Does it work with our nightly builds?
2. What does `/buildconfig` tell?

**Reply by JackZ0** [reply]
this use apt tool

**Comment by JackZ0** [comment]
> When creating a full-screen session to a Windows remote client, the mouse pointer on the Windows remote client isn't displayed. The window is receiving the mouse input (I can see things in the window that have a mouseover event get highlighted if I accidentally move the mouse over it, I just don't see the actual mouse cursor the depicts the mouse cursor location).
> 
> This first started after I upgraded to Ubuntu 21.10 from Ubuntu 20.04. I have re-installed freerdp using:
> 
> sudo apt remove freerdp2-x11 -y sudo apt remove freerdp2-shadow-x11 -y
> 
> sudo apt install freerdp2-x11 -y sudo apt install freerdp2-shadow-x11 -y
> 
> The command I'm using to connect to the Windows remote client is:
> 
> xfreerdp /bpp:32 /u:username /p:password +fonts +window-drag /floatbar:show:always /f /t:"Windows File Server" /v:192.168.0.15
> 
> The above command worked as expected prior to upgrading to Ubuntu 21.10.
> 
> Mark Korenkiewicz

Why not use source code compilation?
